src/engine/ruleset/decoders/fortinet/client-endpoint.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: decoder/fortinet-client-endpoint/0
+
+metadata:
+  title: Decoder for Fortinet FortiClient Endpoint Security
+  description: Decoder for Fortinet FortiClient Endpoint Security
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
```

Comment:
Wazuh Engine . New development.

Reply:
Not sure mentioning 4.3 here

Reply:
In syslog.yml and json.yml, for example, use compatibility section with "4.3". What do you think @NahuFigueroa97?

Reply:
Done